components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -295,6 +300,7 @@ export default class EditPostModal extends React.PureComponent {
                         container={this.getContainer}
                         target={this.getEditPostControls}
                         onHide={this.hideEmojiPicker}
+                        onEmojiClose={this.handleEmojiClose}
```

Comment:
Maybe no need to create another props of `onEmojiClose` as it has similar functionality with the `onHide`.